src/rwe/Unit.cpp

```diff
@@ -128,6 +120,7 @@ namespace rwe
         if (hitPoints + deltaHp >= maxHitPoints)
         {
             hitPoints = maxHitPoints;
+            lifeState = LifeState::Alive;
```

Comment:
I had to add this as repairing using the build logic could invalidate the "isBeingBuilt" check